model_compression_toolkit/core/common/quantization/core_config.py

```diff
@@ -26,19 +25,26 @@ class CoreConfig:
     def __init__(self,
                  quantization_config: QuantizationConfig = QuantizationConfig(),
                  mixed_precision_config: MixedPrecisionQuantizationConfigV2 = None,
-                 debug_config: DebugConfig = DebugConfig()
+                 debug_config: DebugConfig = DebugConfig(),
+                 trace_hessian_cfg: TraceHessianConfig = TraceHessianConfig()
                  ):
         """
 
         Args:
             quantization_config (QuantizationConfig): Config for quantization.
             mixed_precision_config (MixedPrecisionQuantizationConfigV2): Config for mixed precision quantization (optional, default=None).
             debug_config (DebugConfig): Config for debugging and editing the network quantization process.
+            trace_hessian_cfg (TraceHessianConfig): Configuration for the trace hessian service computation.
         """
         self.quantization_config = quantization_config
         self.mixed_precision_config = mixed_precision_config
         self.debug_config = debug_config
+        self.trace_hessian_cfg = trace_hessian_cfg
 
     @property
     def mixed_precision_enable(self):
         return self.mixed_precision_config is not None
+
+    @property
+    def init_hessian_service(self):
```

Comment:
change the name, it sounds like a method that initiate the hessian service

Reply:
also add return type and description

Reply:
We don't use it eventually. I'll remove it.